core/src/main/scala/cromwell/core/labels/Labels.scala

```diff
@@ -1,10 +1,17 @@
 package cromwell.core.labels
 
+import cats.data.Validated._
+import cats.instances.vector._
+import cats.syntax.traverse._
+import lenthall.validation.ErrorOr.ErrorOr
+
 import scala.collection.JavaConverters._
 
 case class Labels(value: Vector[Label]) {
 
-  def asJesLabels = (value map { label => label.key -> label.value }).toMap.asJava
+  def asMap = (value map { label => label.key -> label.value }).toMap
```

Comment:
I normally prefer the postifx form for these things but find when you're adding stuff to the end causing the need to wrap with `()` that it's easier to just go old school (e.g. `value.map(label => ???).toMap`)